packages/core/docs/guide/user-profile.md

```diff
@@ -0,0 +1,203 @@
+import { onSSR } from '@vue-storefront/core';
+
+export default {
+  setup () {
+    const {
+      load,
+      user
+    } = useUser();
+
+    onSSR(async () => {
+      await load();
+    });
+
+    return {
+      load,
+      user
+    }
+  }
+}
+```
+
+Similar to other composables, it's important to remember to not directly access `user` properties, but use `userGetters`. This will ensure that your code is integration-agnostic and proof to (some) future changes.
+
+```js{1,3,17,22}
+import { useUser, userGetters } from '{INTEGRATION}';
+import { onSSR } from '@vue-storefront/core';
+import { computed } from '@vue/composition-api';
+
+export default {
+  setup () {
+    const {
+      load,
+      user
+    } = useUser();
+
+    onSSR(async () => {
+      await load();
+    });
+
+    const firstName = computed(() => userGetters.getFirstName(user.value));
+
+    return {
+      load,
+      user,
+      firstName
+    }
+  }
+}
+```
+
+## Modifying user
+
+Updating user data (except for the current password, which is described in the [Changing password](#changing-password) section) can be done using `updateUser` method in `useUser` composable.
+
+```js
+import { useUser } from '{INTEGRATION}';
+
+export default {
+  setup () {
+    const { updateUser } = useUser();
+
+    const onSubmit = async (formData) => {
+      try {
+        await updateUser({ user: formData });
+        // 'user' property is now updated
+      } catch (error) {
+        // handle update error
+      }
+    };
+
+    return {
+      onSubmit
+    }
+  }
+}
+```
+
+## Changing password
+
+Updating user password can be done using `changePassword` method in `useUser` composable. It requires the current and new password, to confirm user identity.
+
+```js
+import { useUser } from '{INTEGRATION}';
+
+export default {
+  setup () {
+    const { changePassword } = useUser();
+
+    const onSubmit = async (formData) => {
+      try {
+        await changePassword({
+          current: formData.currentPassword,
+          new: formData.newPassword
+        });
+      } catch (error) {
+        // handle update error
+      }
+    };
+
+    return {
+      onSubmit
+    }
+  }
+}
+```
+
+## Managing addresses (billing and shipping)
+
+Managing billing and shipping addresses is done using dedicated [useUserBilling](../composables/use-user-billing.md) and [useUserShipping](../composables/use-user-shipping.md) composables.
+
+For more information, please refer to their documentation.
+
+## Listing user orders
+
+Listing user orders is done using dedicated [useUserOrders](../composables/use-user-orders.md) composable.
+
+For more information, please refer to its documentation.
+
+## Customizing addresses and forms in the user profile
+
+In default theme, components used to display addresses and containing forms for updating user information are blank. This is due to the fact that each integration might use different data formats and properties.
```

Comment:
This paragraph relevant for integration guide, not users guide therefore should be a part of integration guide. Please move it there